ui/src/scenes/Schema/Table/ContextualMenu/services.ts

```diff
@@ -0,0 +1,38 @@
+import * as QuestDB from "utils/questdb"
+import { trim } from "ramda"
+
+export const formatTableSchemaQueryResult = (
+  name: string,
+  result: QuestDB.QueryRawResult,
+): string => {
+  if (result.type === QuestDB.Type.DQL) {
+    let query = `CREATE TABLE ${name} (`
+
+    for (let i = 0; i < result.count; i++) {
+      const column = result.dataset[i]
+      const name = column[0]
+      const typeDef = column[1]
+      const indexed = column[2]
+      const indexBlockCapacity = column[3]
+      const symbolCached = column[4]
+      const symbolCapacity = column[5]
+
+      query += `${name} ${typeDef} `
```

Comment:
Check if the table is from a .csv import, if so, the table name needs to be wrapped in single quotes:

`CREATE TABLE 'my_import.csv'`

Reply:
@bsmth , for this one, simple solution would be to add the quotes all the time, if that does not break the queries not using the quotes 

i.e. CREATE TABLE my_table(symb SYMBOL, price DOUBLE, ts TIMESTAMP, s STRING);
would become CREATE TABLE 'my_table'(symb SYMBOL, price DOUBLE, ts TIMESTAMP, s STRING);

is that reasonable?

Reply:
Yeah this is reasonable